dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/BulkDataRequest.kt

```diff
@@ -0,0 +1,18 @@
+package org.dataland.datalandcommunitymanager.model.dataRequest
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.dataland.datalandbackend.openApiClient.model.DataTypeEnum
+
+/**
```

Comment:
Is there the option a user can follow up on their request, e.g. see how often other users requested the same data, or if the status of the request changed?

Reply:
Currently a user can do the following things:
- see all requests that the user has done so far (endpint GET /requests/user)
- see how ofter others have requested the same data (endpoint GET /requests)

Reply:
Ok, thank you